contributions/api/controllers/contributions.py

```diff
@@ -935,3 +942,49 @@ def admin_reviewers_delete(token_info, id):
         msg_json = jsonutils.create_log_json("Contribution Admin ", "DELETE", msg)
         logging.info("Contribution Admin DELETE " + json.dumps(msg_json))
         return rs_handlers.not_found(msg_json)
+
+
+def send_email_new_reviewer(username):
+    """
+    Method to send email to user for new reviewers
+    Args:
+        username (str) : github username of reviewer
+    """
+    # check if the dataset is existing with given github username
+    dataset = mongoutils.get_reviewers_record(coll_reviewer, username)
+    if dataset is None:
+        msg = {
+            "reason": "Github Username not present in the database: " + str(username),
+            "error": "Bad Request: " + request.url,
+        }
+        msg_json = jsonutils.create_log_json("Contribution Admin", "POST", msg)
+        logging.error("Contribution Admin POST " + json.dumps(msg_json))
+        return rs_handlers.bad_request(msg)
+
+    if 'email' in dataset[0].keys():
+        subject = "Reviewer updated"
+        message = "Reviewer has been updated for a contribution"
+        adminutils.send_email(dataset[0]['email'], subject, message)
+
+def send_email_new_contribution(username):
+    """
+    Method to send email to user for new contribution
+    Args:
+        username (str) : github username of reviewer
+    """
+    # check if the dataset is existing with given github username
+    dataset = mongoutils.get_reviewers_record(username)
+    if dataset is None:
+        msg = {
+            "reason": "Github Username not present in the database: " + str(username),
+            "error": "Bad Request: " + request.url,
+        }
+        msg_json = jsonutils.create_log_json("Contribution Admin", "POST", msg)
+        logging.error("Contribution Admin POST " + json.dumps(msg_json))
+        return rs_handlers.bad_request(msg)
+
+    if 'email' in dataset[0].keys():
+        subject = "New Contribution"
+        message = "New contribution has been added for your review"
+        adminutils.send_email(dataset[0]['email'], subject, message)
+
```

Comment:
what if there is no 'email'? Does it give an error or just return without warning logging or error posting. In theory, this shouldn't happen since the connexion will filter the reviewer record not being posted without email but if the reviewer is manually added and the email is missing or having some typo or something in its json, this might happen

Reply:
added error message if email field not present.